javascript/javascript1/week4/cc_type/cc.js

```diff
@@ -0,0 +1,44 @@
+function getCardType(number) {
+    // visa
+    let re = new RegExp("^4");
+    if (number.match(re) != null)
```

Comment:
you can also do `number.match(re)` without the `null` comparison. it has the same meaning.